data/tribes/initialization/amazons/units.lua

```diff
@@ -1355,14 +1376,26 @@ wl.Descriptions():new_tribe {
                pgettext("amazons_building", "The water gatherer’s hut needs open water within the work area. Your workers cannot dig up water from the ground!"),
                -- TRANSLATORS: Note helptext for an Amazon production site: Water Gatherer's Hut, part 2
                pgettext("amazons_building", "Roads and trees along the shoreline block drawing water."),
+            },
+            performance = {
+               -- TRANSLATORS: Performance helptext for an Amazon production site: Water Gatherer's Hut, part 1
+               pgettext("amazons_building", "This building can produce one bucket of water between %1$s and %2$s on average, depending on how far the carrier has to walk."):bformat(format_seconds(16), format_seconds(32)),
+               -- TRANSLATORS: Performance helptext for an Amazon production site: Water Gatherer's Hut, part 2
+               pgettext("amazons_building", "In case of slopes or obstacles in the way, the time is increased.")
             }
          }
       },
       {
          name = "amazons_rare_tree_cutters_hut",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Rare Tree Cutter's Hut
-            purpose = pgettext("amazons_building", "Harvests rare trees in the surrounding area. Delivers ironwood, balsa and rubber.")
+            purpose = pgettext("amazons_building", "Harvests rare trees in the surrounding area. Delivers ironwood, balsa and rubber."),
+            performance = {
+               -- TRANSLATORS: Performance helptext for an Amazon production site: Rare Tree Cutter's Hut, part 1
+               pgettext("amazons_building", "This building can produce one of each rare wood between %1$s and %2$s, depending on how far the woodcutter has to walk."):bformat(format_minutes_seconds(2, 22), format_minutes_seconds(4, 20)),
```

Comment:
This holds when all 3 are available, but if only 1 or 2, then those are faster. I suggest something like:
"… can produce a rare wood between 47s and 1m27s depending on how far the master woodcutter has to walk. If all kinds of rare tree are available within her work area, then one of each kind is produced between 2m22s and 4m20s."

Reply:
This is good idea - similar style like in https://github.com/widelands/widelands/pull/5861
I used similar wording also for Hunter-Gatherer's Hut, to prevent confusion.